config/initializers/cors.rb

```diff
@@ -5,7 +5,7 @@
 # Ref: https://stackoverflow.com/questions/56960709/rails-font-cors-policy
 Rails.application.config.middleware.insert_before 0, Rack::Cors do
   allow do
-    origins '*'
+    origins 'studio.datachain.ai,chatwoot.studio.datachain.ai,studio.dvc.dev'
```

Comment:
@yathomasi, the syntax was wrong, and, as per https://github.com/cyu/rack-cors/issues/178, it should look like this:

```suggestion
    origins 'https://studio.datachain.ai', 'https://chatwoot.studio.datachain.ai', `https://studio.dvc.dev'
```

Moreover, is this change necessary?

Reply:
Yeah, it might not have been necessry #2 was required. Should we only implement that for ease of use ?